docs/whatsnew/5.2.rst

```diff
@@ -12,7 +12,26 @@ the 5.1 release.
 
 In particular, this release includes:
 
-* nothing yet
+* :ref:`whatsnew-5.2-cosmology`
+
+
+.. _whatsnew-5.2-cosmology:
+
+Updates to ``Cosmology``
```

Comment:
```suggestion
Updates to |Cosmology|
```

Reply:
Thanks, however it's even more general...
I'll do: 

```
Updates to `astropy.cosmology`
==============================
```